src/main.c

```diff
@@ -0,0 +1,151 @@
+
+typedef unsigned char u8;
+typedef unsigned short u16;
+typedef unsigned long u32;
+typedef signed char s8;
+typedef signed short s16;
+typedef signed long s32;
+
+struct Unk2106FA0
+{
+  s32 unk0;
+  s32 unk4;
+  s32 unk8;
+  s32 unkC;
+  s32 unk10;
+  s32 unk14;
+  s32 unk18;
+  s32 unk1C;
+  s32 unk20;
+};
+
+struct Unk21C48B8
+{
+  void (*unk0)(s32);
+  s32 unk4;
+  s32 unk8;
+  s32 unkC;
+  s32 unk10;
+  s32 unk14;
+  s32 unk18;
+  s32 unk1C;
+  s32 unk20;
+  s32 unk24;
+  s32 unk28;
+  s32 unk2C;
+  s32 unk30;
+  s32 unk34;
+  s32 unk38;
+  u8 filler3C[0xC];
+  s32 unk48;
+  u8 filler4C[0x20];
+  s32 unk6C;
+};
+
+extern struct Unk2106FA0 gUnknown2106FA0;
+extern struct Unk2106FA0 gUnknown2106FA0_2; // same as the first one, it's referenced twice in the constant pool...
+extern struct Unk2106FA0 gUnknown2106FB8;
+extern struct Unk21C48B8 gUnknown21C48B8;
+
+extern s32 gUnk027FFC20;
+extern struct { 
+  s32 unk0;
+  s32 unk4;
+  u8 unk8;
+} gUnk021C4918;
+
+extern void FUN_020D50D8();
+extern void FUN_020C9C0C();
+extern void FUN_02016438(s32);
+
+extern const s32 CONST_3F; // 0x0000003F
+extern const s32 CONST_34; // 0x00000034
+
+extern struct Unk21DBE18 gUnk021DBE18; 
+extern struct Unk21DBE18 gUnk021D76C8;
+
+
+void NitroMain(void)
+{
+	FUN_02016014();
```

Comment:
tabs to spaces (tab = 4 spaces)

Reply:
can we get an editorconfig to enforce that pls?